IdentityCore/src/oauth2/token/MSIDPrimaryRefreshToken.m

```diff
@@ -29,6 +29,7 @@
 #import "MSIDAuthority.h"
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->